### PR TITLE
add system info cli command

### PR DESCRIPTION
**Pull request by thedavidprice**
**Notes/Questions:**
- ESlint warning for 'envinfo' import: anything I should/can do about this?
- open to suggestions about option name/alias of "clipboard" and "C"
- automatic "copy to clipboard" is nice, but clipboardy is a beast of an import and I'm mixed on whether this is necessary
- re: Info Output
    - many references include output for 'System: CPU' -- will we need this?
    - I added 'Databases: SQLite' -- since this is handled by Prisma2, is it helpful here?

### CODE REVIEW

**Comment by thedavidprice** [comment]
Closes #321

**Comment by peterp** [comment]
> ESlint warning for 'envinfo' import: anything I should/can do about this?

Can you let me know what the import error is?

> automatic "copy to clipboard" is nice, but clipboardy is a beast of an import and I'm mixed on whether this is necessary

I personally don't think we should mess with people's clipboards.

> re: Info Output

I think what you have is perfect.

**Comment by satyarohith** [comment]
The output is only copied when a user deliberately passes the flag, right? I think it's handy.

**Comment by weaversam8** [comment]
I think this is in great shape!

> automatic "copy to clipboard" is nice, but clipboardy is a beast of an import and I'm mixed on whether this is necessary

Though it's a nice feature, I lean towards the idea of keeping things lean. It isn't that difficult to copy something to clipboard manually.

> re: Info Output

Looks great! Do we want to add printing out the version of `prisma2` or `@prisma/client`? I know we're supposed to be pinned on an internal version, but just in case somebody tried to install their own in the future, might be helpful just to verify we're running the same version.

**Comment by thedavidprice** [comment]
Thanks, all. I’ll get back to these comments later today.

Realized I never added the current output for feedback:

File: packages/cli/src/commands/info.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+// and gridsome/packages/cli/lib/commands/info.js
+import envinfo from 'envinfo'
+import clipboardy from 'clipboardy'
+
+export const command = 'info'
+export const desc = 'Get environment information'
+export const builder = {
+  clipboard: {
+    alias: `C`,
+    type: `boolean`,
```
[the remark this answers is not in the document]
Comment (thedavidprice) [reply packages/cli/src/commands/info.js]: Ah, good catch... understood and thought I had already swapped those.

File: packages/cli/src/commands/info.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    envinfo
+      .run({
+        System: ['OS', 'Shell'],
+        Binaries: ['Node', 'Yarn'],
+        Browsers: ['Chrome', 'Edge', 'Firefox', 'Safari'],
+        // yarn workspaces not supported :-/
+        npmPackages: ['@redwoodjs/core'],
+        Databases: ['SQLite'],
+      })
+      .then((envinfoOutput) => {
```
[the remark this answers is not in the document]
Comment (thedavidprice) [reply packages/cli/src/commands/info.js]: Updated to use async/await. The `const output = ...` didn't seem necessary here. But I could be missing something obvious; just let me know.

**Comment by thedavidprice** [comment]
Thanks for the feedback and help here, everyone. Latest version changes:

- cleaned up the code ('cause I did a late-night copy/paste and it needed some love!)
- I vetoed the "automatic copy" myself. It seems unnecessary in practice. And we'll find out either way if it's needed based on future use. Easy to add now that we have an example in the branch.

@peterp 
**Import ESLint Warning**
We have this occurring elsewhere and I believe it more about the modules TS support more than anything else. So not actionable. But for fyi:
```javascript
Could not find a declaration file for module 'envinfo'. '/Users/price/Repos/redwoodjs-redwood/node_modules/envinfo/dist/envinfo.js' implicitly has an 'any' type.
  Try `npm install @types/envinfo` if it exists or add a new declaration (.d.ts) file containing `declare module 'envinfo';`ts(7016)
```

File: packages/cli/src/commands/info.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+export const command = 'info'
+export const desc = 'Prints your system environment information'
+export const builder = {
+  clipboard: {
+    alias: 'C',
+    type: 'boolean',
+    default: false,
+    describe: 'Copy info to clipboard',
+  },
+}
```
Comment (weaversam8) [inline packages/cli/src/commands/info.js]:
```suggestion
```

File: packages/cli/package.json [hunk 4]
```diff
@@ -16,9 +16,11 @@
     "@types/react": "^16.9.19",
     "camelcase": "^5.3.1",
     "chalk": "^3.0.0",
+    "clipboardy": "^2.3.0",
```
Comment (weaversam8) [inline packages/cli/package.json]:
```suggestion
```

File: yarn.lock [hunk 5]
```diff
@@ -4709,6 +4714,15 @@ cli-width@^2.0.0:
...
 
+clipboardy@^2.3.0:
+  version "2.3.0"
+  resolved "https://registry.yarnpkg.com/clipboardy/-/clipboardy-2.3.0.tgz#3c2903650c68e46a91b388985bc2774287dba290"
+  integrity sha512-mKhiIL2DrQIsuXMgBgnfEHOZOryC7kY7YO//TN6c63wlEm3NG5tz+YgY5rVi29KCmq/QQjKYvM7a19+MDOTHOQ==
+  dependencies:
+    arch "^2.1.1"
+    execa "^1.0.0"
+    is-wsl "^2.1.1"
+
```
Comment (weaversam8) [inline yarn.lock]:
```suggestion
```

File: packages/cli/src/commands/info.js (hunk 3 above)
Comment (thedavidprice) [reply packages/cli/src/commands/info.js]: 🤦‍♂ good catch… I’m moving to fast across many things today.Thanks!

File: yarn.lock (hunk 5 above)
Comment (thedavidprice) [reply yarn.lock]: done and cleaned up yarn.lock as well

**Comment by thedavidprice** [comment]
Note: some updates to our ESLint/Prettier config were causing Lint check error. I committed simple fix to master here bd98d07

**Pull request merged**